platform/install/helm.mdx

```diff
@@ -120,6 +120,19 @@ ingress:
   enabled: true
   host: vcluster-platform.mytld.com
 ```
+- `env` – Defines environment variables for the vCluster platform components. By default, this field is empty.
+A common use case is configuring network access between the vCluster platform and managed vCluster to allow API communication.
```

Comment:
Use 'managed virtual cluster' instead of 'managed vCluster' per terminology guidelines - vCluster is the product, virtual cluster is what it creates
```suggestion
A common use case is configuring network access between the vCluster platform and managed virtual cluster to allow API communication.
```